bin/codemods/src/assign-component-to-displayname-variable.js

```diff
@@ -50,91 +39,73 @@ const displayNamePropertyIdentifier = {
 
 const isValidComponentString = string => string === 'Component' || string === 'PureComponent';
 
-const argIsCreateClassInstance = arg =>
-	arg.get( 'callee', 'object', 'name' ).value === 'React' &&
-	arg.get( 'callee', 'property', 'name' ).value === 'createClass';
-
-const argIsExtendsComponentInstance = arg =>
-	arg.get( 'type' ).value === 'ClassExpression' &&
-	( isValidComponentString( arg.get( 'superClass', 'name' ).value ) ||
-		( arg.get( 'superClass', 'object', 'name' ).value === 'React' &&
-			isValidComponentString( arg.get( 'superClass', 'property', 'name' ).value ) ) );
-
-const argIsComponent = arg =>
-	argIsCreateClassInstance( arg ) || argIsExtendsComponentInstance( arg );
-
-const getMatchingArg = args => args.filter( argIsComponent )[ 0 ];
+const isCreateClassComponent = arg =>
+	_.get( arg, [ 'callee', 'object', 'name' ] ) === 'React' &&
+	_.get( arg, [ 'callee', 'property', 'name' ] ) === 'createClass';
+
+const isEs6ClassComponent = arg =>
+	_.get( arg, 'type' ) === 'ClassExpression' &&
+	( isValidComponentString( _.get( arg, [ 'superClass', 'name' ] ) ) ||
+		( _.get( arg, [ 'superClass', 'object', 'name' ] ) === 'React' &&
+			isValidComponentString( _.get( arg, [ 'superClass', 'property', 'name' ] ) ) ) );
```

Comment:
Sometimes I disagree with prettier about what is 'pretty' :P ( `] ) ) ) )` )

Reply:
I agree its not pretty.  I think another takeaway could be to refactor it into multiple lines